src/commands/Utility/prefix.ts

```diff
@@ -34,7 +34,7 @@ const command: Command = {
 	async execute(interaction, getString: GetStringFunction) {
 		if (!interaction.inCachedGuild()) return
 		const randomTip = generateTip(getString),
-			nickNoPrefix = interaction.member.displayName.replaceAll(/\[[^\s]*\] ?/g, "").trim(),
+			nickNoPrefix = interaction.member.displayName.replaceAll(/\[[^.]*\] ?/g, "").trim(),
```

Comment:
What?
```suggestion
			nickNoPrefix = interaction.member.displayName.replaceAll(/\[.*\] ?/g, "").trim(),
```

Reply:
Seems like the old regex was weird and I kept that weird part (still worked tho ???)

Reply:
It wasn’t weird. Your change meant that this would never match regardless of what the prefix had (negated set with every character in it)